backbone/src/main/java/org/researchstack/backbone/ui/ConsentViewTaskActivity.java

```diff
@@ -187,65 +188,66 @@ private String getSignatureHtmlContent(@Nullable String completeName, @NonNull S
     {
         StringBuffer body = new StringBuffer();
 
-        String hr = "<hr align='left' width='100%' style='height:1px; border:none; color:#000; background-color:#000; margin-top: -10px; margin-bottom: 0px;' />";
+        String hr = "<hr align='left' width='100%' style='height:1px; border:none; color:#000; background-color:#000; " +
+                "margin-top: 5px; margin-bottom: 0px;'/>";
 
-        String signatureElementWrapper = "<p><br/><div class='sigbox'><div class='inbox'>%s</div></div>%s%s</p>";
+        String signatureElementWrapper = "<div class='sigbox'><div class='inbox'>%s</div></div>%s%s";
         String imageTag = null;
 
         List<String> signatureElements = new ArrayList<>();
 
-        if(role == null)
-        {
+        if(role == null) {
             throw new RuntimeException("Consent role cannot be empty");
         }
 
         // Signature
-        if (completeName != null)
-        {
+        if (completeName != null) {
             String base = getString(R.string.rsb_consent_doc_line_printed_name, role);
             String nameElement = String.format(signatureElementWrapper, completeName, hr, base);
             signatureElements.add(nameElement);
         }
 
-        if (signatureB64 != null)
-        {
-            imageTag = "<img width='100%%' alt='star' src='data:image/png;base64,"+signatureB64+"' />";
+        if (signatureB64 != null) {
+            imageTag = "<img width='100%' alt='star' style='max-height:100px;max-width:200px;height:auto;width:auto;' " +
```

Comment:
nit: I don't like this long string split (it may have been "auto-format" but I'm not sure if it's better or worse; feel free to leave as-is if you think it's ok.